src/main/java/edu/umd/lib/process/WufooProcessor.java

```diff
@@ -58,18 +58,34 @@ public WufooProcessor() {
   public void process(Exchange exchange) throws Exception {
 
     String message = exchange.getIn().getBody(String.class);
-    Map<String, List<String>> parameters = getQueryParams(message);
 
+    Map<String, List<String>> parameters = getQueryParams(message);
     checkHandshake(parameters);
+    String formName = getFormName(parameters.get("FormStructure"));
     Map<String, String> fields = getFields(parameters);
     JSONArray fieldsList = getFieldStructure(parameters.get("FieldStructure"), fields);
     HashMap<String, String> values = extractParameters(fieldsList);
+    values.put("FormName", formName);
 
     exchange.getOut().setBody(values);
     log.info("Total Number of Parameters from the request:" + parameters.size());
 
-    // SysAidConnector sysaid = new SysAidConnector();
-    // sysaid.createServiceRequest(values);
+  }
+
+  /****
+   * Get Form Name from the Wufoo Request
+   *
+   * @param formStructureArray
+   * @return
+   */
+  public String getFormName(List<String> formStructureArray) {
+    try {
+      JSONObject formStructure = new JSONObject(formStructureArray.get(0).toString());
+      return formStructure.getString("Name");
```

Comment:
This Display Name is on the form. And Wufoo Form Administrator can change this value.